spec/models/video_spec.rb

```diff
@@ -1,11 +1,37 @@
 require 'spec_helper'
 
 describe Video do
-  it "saves itself" do
-    video = Video.new(title: "Test title", description: "A fish and ham sandwich", small_cover_url: "/public/tmp/monk.jpg", large_cover_url: "/public/tmp/monk_large.jpg")
-    video.save
-    #Video.first.title.should == "Test title"
-    expect(Video.first).to eq(video)
+  it { should belong_to(:category)}
+  it { should validate_presence_of(:title)}
+  it { should validate_presence_of(:description)}
+
+  describe "search_by_title" do
```

Comment:
RSpecs convention is `.` for class method and `#` for instance methods - So here I recommend `describe ".search_by_title" do`. Check this guide off - http://www.betterspecs.org